src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/SetAzureRmDataLakeAnalyticsAccount.cs

```diff
@@ -30,35 +30,38 @@ public class SetAzureDataLakeAnalyticsAccount : DataLakeAnalyticsCmdletBase
 
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = false,
             HelpMessage =
-                "Name of the new Data Lake account to set as the default storage for this DataLakeAnalytics account.")]
-        [ValidateNotNullOrEmpty]
-        public string DefaultDataLakeStore { get; set; }
```

Comment:
Was never supported. Same comment as above. We were lucky no one tried to use this option, since it would have always resulted in an exception from the server saying "changes to default store are not allowed"